src/components/App.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+
+import Title from './Title';
+import CustomerReviews from './customerReviews';
+import PriceBox from './PriceBox';
+import Stock from './Stock';
+import ItemDescription from './ItemDescription';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    //  sample Data
+    this.state = {
+      data: [{
+        'Brand Name': 'Oxygen',
+        'Description 1': 'Donec dapibus. Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante.',
+        'Description 2': 'Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque. Quisque porta volutpat erat. Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla.',
+        'Description 3': 'Nam dui. Proin leo odio, porttitor id, consequat in, consequat ut, nulla.',
+        'Description 4': 'In hac habitasse platea dictumst. Etiam faucibus cursus urna.',
+        'Description 5': 'Aliquam quis turpis eget elit sodales scelerisque. Mauris sit amet eros.',
+        'Inventory Amount': 0,
+        'Is Customer Prime Member?': false,
+        // currently Prime Member does nothing. All shows as Prime
```

Comment:
I think this would be a great addition to the description module! Speaking with you about this in person, I think this could be a stretch goal once we finish our proxy and once you finish other priorities on your list. I think a top-down state management for the Prime member would be the easiest way to approach this so you don't encounter the re-rendering of other components as we discussed earlier.